libnetwork/drivers/bridge/setup_device.go

```diff
@@ -16,8 +16,10 @@ import (
 // SetupDevice create a new bridge interface/
 func setupDevice(config *networkConfiguration, i *bridgeInterface) error {
 	// We only attempt to create the bridge when the requested device name is
-	// the default one.
-	if config.BridgeName != DefaultBridgeName && config.DefaultBridge {
+	// the default one. The env var DOCKER_TEST_CREATE_DEFAULT_BRIDGE should be
+	// used only for test purpose.
+	if _, ok := os.LookupEnv("DOCKER_TEST_CREATE_DEFAULT_BRIDGE"); !ok &&
```

Comment:
From our earlier conversation on Slack; could we use the env-var to override the _name_ of the bridge to use (i.e., if it's set (non-empty value) to override `DefaultBridgeName`, and consider it to be the default bridge name)?